src/main/java/com/alkemy/ong/service/SlideService.java

```diff
@@ -41,4 +51,32 @@ public void deleteSlide(Long idSlide) throws SlideNotFoundException {
             throw new SlideNotFoundException(String.format("Slide with id: %s not found", idSlide));
         }
     }
+
+    @Transactional
+    public Slide createSlide(Slide slide) {
+        SlideModel slideModel = SlideMapper.mapDomainToModel(slide);
+        Optional<OrganizationModel> organizationModelOptional = organizationRepository.findById(slideModel.getOrganization().getId());
+        if (organizationModelOptional.isEmpty()) {
+            throw new OrganizationNotFoundException(String.format("Organization with ID: %s not found", slideModel.getOrganization().getId()));
+        }
+        slideModel.setOrganization(OrganizationMapper.mapDomainToModel(slide.getOrganization()));
```

Comment:
Deberías hacer:
```suggestion
        slideModel.setOrganization(organizationModelOptional.get());
```
ya que por eso la vamos a buscar primero 👌🏼